coral/src/app/router.tsx

```diff
@@ -223,6 +224,10 @@ const routes: Array<RouteObject> = [
         path: Routes.TOPIC_SCHEMA_REQUEST,
         element: <SchemaRequest />,
       },
+      {
```

Comment:
This should go behind the feature flag for topic details so it's not accessible until we official release the feature.